app/controllers/api/v4/patient_controller.rb

```diff
@@ -7,10 +10,7 @@ def activate
     patient  = passport&.patient
     return head :not_found unless patient.present? && patient.latest_mobile_number.present?
```

Comment:
Should `patient.latest_mobile_number.present?` throw a different error than `not_found`? That way the patient can be informed that they need to ask their nurse/doctor to update their number.

Reply:
We discussed this and we may do that later. For now we're keeping it simple.